package.json

```diff
@@ -722,8 +724,6 @@
     }
   },
   "scripts": {
-    "build:tsc": "tsc -b",
-    "watch:tsc": "tsc -b -w",
```

Comment:
Removing the TSC component of the build appears to remove type checking from the build entirely. I'd prefer to have script option that does perform type checking, though I don't oppose having a build option that does not require type checking, in addition.

However, if we remove the `tsc` components of the build, then the `build` and `watch` scripts can be simplified considerably: no need for the separate `build:esbuild` script or the `pre-watch` and `do-watch` scripts.